test/autoprefixer.test.js

```diff
@@ -422,6 +422,7 @@ describe('hacks', () => {
     it('supports text-decoration',      () => test('text-decoration'));
     it('ignores modern direction',      () => test('animation'));
     it('supports overscroll-behavior',  () => test('overscroll-behavior'));
+    it('supports webkit prefixes',      () => test('color-adjust'));
```

Comment:
Is the name of test is ok? I tried to get the correct naming logic from the test cases above.

Reply:
Nope, the test name is incorrect. Check others, they contain hack name, not prefix.